src/azure-cli/azure/cli/command_modules/sql/custom.py

```diff
@@ -3115,12 +3117,26 @@ def update_long_term_retention(
     if yearly_retention and not week_of_year:
         raise CLIError('Please specify week of year for yearly retention.')
 
-    if make_backups_immutable:
-        confirmation = prompt_y_n("""Immutable LTR backups can't be changed or deleted.
-         You'll be charged for LTR backups for the full retention period.
-         Do you want to proceed?""")
-        if not confirmation:
-            return
+    if time_based_immutability and time_based_immutability.lower() == "enabled":
+        if not yes:
+            confirmation = prompt_y_n("""Immutable LTR backups can't be changed or deleted.
+            You'll be charged for LTR backups for the full retention period.
+            Do you want to proceed?""")
+            if not confirmation:
+                return
+
+    if time_based_immutability.lower() == "true":
+        time_based_immutability = "Enabled"
+
+    if time_based_immutability.lower() == "false":
+        time_based_immutability = "Disabled"
```

Comment:
```suggestion
    if time_based_immutability and time_based_immutability.lower() == "true":
        time_based_immutability = "Enabled"

    if time_based_immutability and time_based_immutability.lower() == "false":
        time_based_immutability = "Disabled"

    if time_based_immutability and time_based_immutability.lower() == "enabled":
        if not yes:
            confirmation = prompt_y_n("""Immutable LTR backups can't be changed or deleted.
            You'll be charged for LTR backups for the full retention period.
            Do you want to proceed?""")
            if not confirmation:
                return
```
If `time_based_immutability` is true, there should be a prompt as well, right?

Reply:
yes